app/DoctrineMigrations/Version20190708122957.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Wallabag\CoreBundle\Doctrine\WallabagMigration;
+
+/**
+ * Change reading speed value.
+ */
+final class Version20190708122957 extends WallabagMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('UPDATE ' . $this->getTable('config', true) . ' SET reading_time = reading_time*200;');
```

Comment:
I'm not sure if it's an issue or not, but `;` isn't required at the end of SQL statements called like this.